warehouse/internal/repo/table_upload.go

```diff
@@ -112,8 +112,22 @@ func (repo *TableUploads) GetByUploadID(ctx context.Context, uploadID int64) ([]
 	if err != nil {
 		return nil, fmt.Errorf("querying table uploads: %w", err)
 	}
+	defer func() { _ = rows.Close() }()
 
-	return repo.parseRows(rows)
+	var tableUploads []model.TableUpload
+	for rows.Next() {
+		var tableUpload model.TableUpload
+		err := scanTableUpload(rows.Scan, &tableUpload)
+		if err != nil {
+			return nil, fmt.Errorf("parsing rows: %w", err)
```

Comment:
Would more context help here? Perhaps `get by upload id: parsing rows: %w`.

Reply:
Adding appropriate information about the action we are performing.